src/com/opc/opcconnectionimpl.cpp

```diff
@@ -152,6 +163,30 @@ bool COpcConnectionImpl::addGroup(const char* paGroupName, unsigned long paReqUp
   return true;
 }
 
+void COpcConnectionImpl::removeItems(const char* paGroupName){
+  if(nullptr == paGroupName){
+   DEVLOG_INFO("COpcConnectionImpl::removeItems: group name is nullptr\n");
+   return;
+  }
+  DEVLOG_INFO("removing items in COpcConnectionImpl[%s]\n", paGroupName);
+  std::vector<COPCItem *> lItems;
+  TOpcItemsIt it;
```

Comment:
Could you please combine 173 and 175 into a combined declaration/definition statement?